marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/map-multiple-entities/runMapping.sjs

```diff
@@ -0,0 +1,107 @@
+const mapping = require("/data-hub/5/builtins/steps/mapping/entity-services/main.sjs");
+const test = require("/test/test-helper.xqy");
+
+let assertions = [];
+function runMapping () {
+  let doc = fn.head(cts.doc('/content/customerInfo.json'));
+  let result = mapping.main({uri: '/content/customerInfo.json', value: doc}, {
+    mapping: {name: 'mapCustomersJSON'},
+    outputFormat: 'json'
+  });
+  let instance0 = result[0].value.root.envelope.instance;
+  let instance1 = result[1].value.root.envelope.instance;
+  let instance2 = result[2].value.root.envelope.instance;
+  let instance3 = result[3].value.root.envelope.instance;
+  let instance4 = result[4].value.root.envelope.instance;
+  let instance5 = result[5].value.root.envelope.instance;
+
+  try {
+    assertions = assertions.concat([
+      test.assertEqual(202, fn.number(instance0.Customer.customerId)),
+      test.assertEqual("Cynthia", fn.string(instance0.Customer.firstName)),
+      test.assertEqual('Waters', fn.string(instance0.Customer.lastName))
+    ]);
+  } catch (e) {
+    assertions.push(
```

Comment:
This seems unusual - is the problem it's trying to solve that you can't see the instance that failed? If so, I think the better way to do it is this (pseudocoding):

```
const message = "instance: " + xdmp.describe(instance0);
test.assertEqual(202, fn.number(...), "customerId is not 202 in " + message);
```

That way you know exactly which assertion failed, along with why, and what the instance actually is.